WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -0,0 +1,57 @@
+
+## Daily routine
+
+1. Start
+2. Check code review requests. [link](https://github.com/zkSNACKs/WalletWasabi/pulls) __Reviews / Awaiting review from you__
+2. Check [priority issues!](https://github.com/zkSNACKs/WalletWasabi/pulls?q=is%3Aopen+is%3Apr+label%3Apriority). Take it or set Assignees and Labels if you can.
+3. Check your own PR-s and answer if neccessary. [link](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+user%3AzkSNACKs+) __Click Created__
```

Comment:
I have 68 here. You suggest me to check all 86 daily?

Reply:
Since he said PRs I think the link should be https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+archived%3Afalse+user%3AzkSNACKs+
Note: You get all the PRs from other zkSNACKs repos.

Reply:
Good!